packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-pwd-recovery-mfa/nodejs/integrationsteps.md

```diff
@@ -1 +1,95 @@
+
+```JavaScript
+router.post('/select-authenticator', async (req, res, next) => {
+  const { idxMethod } = req.getFlowStates();
+  const { authenticator } = req.body;
+  const authClient = getAuthClient(req);
+  const transaction = await authClient.idx[idxMethod]({ authenticator });
+  handleTransaction({ req, res, next, authClient, transaction });
+});
+```
+
+### Step 4: Create code verification page
+
+After the call to `idx.recoverPassword` has responded with a status of `Idx.Status:PENDING`, the user is redirected to a code verification page. The page must display a field to enter a code and a button or link to send the code to Okta for the email verification.
+
+After the user checks their email for the code and enters the value into the field, they click a button to initiate the verification. Call `idx.recoverPassword` with the verification code passed in, as shown in the `authenticator.js` page of the SDK sample application.
+
+```JavaScript
+router.post('/challenge-authenticator/email', async (req, res, next) => {
+  const { idxMethod } = req.getFlowStates();
+  const { verificationCode } = req.body;
+  const authClient = getAuthClient(req);
+  const transaction = await authClient.idx[idxMethod]({ verificationCode });
+  handleTransaction({ req, res, next, authClient, transaction });
+});
+```
+
+If the `idx.recoverPassword` call is successful, it returns a status of `Idx.Status:PENDING` and the `nextStep` field indicates an input of a password value parameter. This status indicates that the user can now change their password. At this point, the user is redirected to the change password page.
+
+### Step 5: Create change password page
+
+Create a change password page that allows the user to enter the new password and initiate the update.
+
+The final step is to make a call to `idx.recoverPassword`, passing in the value of the password, input by the user, as shown in the `recover-password.js` file of the SDK sample application. This call resets the user's password.
+
+```JavaScript
+router.post('/reset-password', async (req, res, next) => {
+  const { password, confirmPassword } = req.body;
+  if (password !== confirmPassword) {
+    next(new Error('Password not match'));
+    return;
+  }
+
+  const authClient = getAuthClient(req);
+  const transaction = await authClient.idx.recoverPassword({ password });
+  handleTransaction({ req, res, next, authClient, transaction });
+});
+```
+
+If the response to the `idx.recoverPassword` call returns `Idx.Status:SUCCESS` and tokens, the change password flow completed successfully and you can redirect the user to the default home page.
```

Comment:
```suggestion
If the response to the `idx.recoverPassword()` call returns `Idx.Status:SUCCESS` and tokens, the change password flow completed successfully and you can redirect the user to the default home page.
```

Reply:
As per other comment on this terminology.